package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "web3-onboard-monorepo",
-  "version": "2.22.0",
+  "version": "2.23.0-alpha.1",
```

```suggestion
  "version": "2.22.0",
```
This is the mono-repo version, for this change to be active the version within the packages/injected/package.json needs the version bump with alpha flag